libs/icu/Makefile

```diff
@@ -41,23 +41,36 @@ define Package/icu
   DEPENDS:=+libstdcpp +libpthread
 endef
 
+CONFIGURE_ARGS_ICU := \
+	--disable-debug \
+	--enable-release \
+	--enable-shared \
+	--enable-static \
+	--enable-draft \
+	--enable-renaming \
+	--disable-tracing \
+	--disable-extras \
+	--enable-dyload \
+	--enable-layout \
+	--enable-layoutex \
```

Comment:
Is there some kind of syntax checker to spot things like this as there is no error or warning when building?